structuralcodes/materials/concrete/_concrete.py

```diff
@@ -63,14 +66,44 @@ def constitutive_law(self) -> ConstitutiveLaw:
         return self._constitutive_law
 
     @constitutive_law.setter
-    def constitutive_law(self, constitutive_law: ConstitutiveLaw) -> None:
-        """Setter for constitutive law."""
-        if 'concrete' in constitutive_law.__materials__:
-            self._constitutive_law = constitutive_law
+    def constitutive_law(
+        self,
+        constitutive_law: t.Union[
+            ConstitutiveLaw,
+            t.Literal['elastic', 'parabolarectangle', 'sargin', 'popovics'],
+        ],
+    ) -> None:
+        """Setter for constitutive law.
+
+        Args:
+        consitutive_law (ConstitutiveLaw): a valid ConstitutiveLaw object
+                for concrete (optional)
+        constitutive_law_name (str): a string defining a valid constitutive law
+                type for concrete.
+                (valid options: 'elastic', 'parabolarectangle',
+                'bilinearcompression', 'sargin', 'popovics').
```

Comment:
Please update the docstring so that it matches the function signature.

Reply:
You right! I forgot to update this! Done i n16967e2fe4a6290daab9551cb04ae7167cc69c21